shm/Item.cpp

```diff
@@ -19,13 +19,28 @@ Cargo& Item::operator-=(const size_t amount) {
 }
 
 bool Item::operator==(const Cargo& cargo) const {
-    return getName() == cargo.getName() &&
+    return getName() == cargo.getName();/* &&
            getBasePrice() == cargo.getBasePrice() &&
-           getPrice() == cargo.getPrice();
+           getPrice() == cargo.getPrice();*/
 }
 
 bool Item::operator!=(const Cargo& cargo) const {
-    return getName() != cargo.getName() ||
+    return getName() != cargo.getName();/* ||
            getBasePrice() != cargo.getBasePrice() ||
-           getPrice() != cargo.getPrice();
+           getPrice() != cargo.getPrice();*/
+}
+
+std::string Item::rarityToString(Rarity rarity) {
+    switch (rarity) {
+    case Rarity::common:
+        return "common";
+    case Rarity::rare:
+        return "rare";
+    case Rarity::epic:
+        return "epic";
+    case Rarity::legendary:
+        return "legendary";
+    default:
+        return "";
```

Comment:
I would go with uppercase

Reply:
You mean first letter uppercase or whole string uppercase?